osu.Android/OsuGameAndroid.cs

```diff
@@ -65,6 +67,13 @@ public override Version AssemblyVersion
             }
         }
 
+        /// <summary>
+        /// Schedules a file to be imported once the game is loaded.
+        /// </summary>
+        /// <param name="stream">A stream to the file to import.</param>
+        /// <param name="filename">The filename of the file to import.</param>
+        public void ScheduleImport(Stream stream, string filename) => WaitForReady(() => this, _ => Task.Run(() => Import(stream, filename)));
```

Comment:
Is there a reason we can't just make this standard for the import logic? ie. making `OsuGameBase.Import` virtual and overriding with the `WaitForReady` addition in `OsuGame`.

Reply:
This one is still standing. Was there a reason for not changing the way this is done?